starport-lambda/src/main/scala/com/krux/starport/lambda/SubmitHandler.scala

```diff
@@ -33,31 +32,42 @@ class SubmitHandler extends RequestHandler[SubmitRequest, SubmitResponse] with L
     var outString = ""
     var errString = ""
 
-    try {
-      SubmitPipeline.main(input.getArgs)
-    } catch {
-      case caughtExit: LambdaExitException => {
-        status = caughtExit.status
-        logger.error("exit:", caughtExit)
-        logger.error(scanTmpFiles())
-      }
-      case unhandled: Throwable =>  {
+    val args: Array[String] = input.getArgs
+
+    args.head match {
+      case "deleteTmpDir" => {
```

Comment:
this one should be in the `try` part below

Reply:
Ah yes, sorry.